linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_sle
```

Comment:
Quoting @ggbecker: This is definitely a problem. The rule.yml uses package installed template. This should either be reverted or the file should be split into two: sle12.yml and sle15.yml